### PR TITLE
B/int/image scheduler

### PR DESCRIPTION
**Pull request by uglrl**
Make sure, you have done de following before merging this pull request:
- [x] You ran `cargo format` to format your code.
- [ ] You ran `cargo clippy` to check for improvements.

### CODE REVIEW

File: backend/tests/image_review_scheduler.rs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            database::factory::DataAccessFactory, flickr_api::flickr_api_handler::FlickrApiHandler,
+        },
+        logic::image_review::image_reviewer::ImageReviewer,
+        trigger::scheduling::scheduler::{ScheduleInfo, Scheduler},
+    },
+    startup::config::ConfigReader,
+};
+use tokio::signal::ctrl_c;
+
+const NEVER: &str = "* * * 31 2 *";
```
Comment (worldofjoni) [inline backend/tests/image_review_scheduler.rs:15]: xD

File: backend/tests/image_review_scheduler.rs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        image_review_schedule: "0 */5 * * * *".to_string(),
+    };
+    let reader = ConfigReader::default();
+    let database_factory = DataAccessFactory::new(reader.read_database_info().unwrap(), false)
+        .await
+        .unwrap();
+    let image_review_data = database_factory.get_image_review_data_access();
+    let image_hoster = FlickrApiHandler::new(reader.read_flickr_info().unwrap());
+    let image_scheduling = ImageReviewer::new(image_review_data, image_hoster);
+    Scheduler::new(info, image_scheduling, MealplanManagerMock::default()).await
```
Comment (worldofjoni) [inline backend/tests/image_review_scheduler.rs:39]: My idea was thath we can probably put the tests for all scheduling in a single test file as it is only a single component (we then can still create multiple test cases). Then we would not need a mock.

**Review by worldofjoni — CHANGES_REQUESTED** [review]

**Comment by worldofjoni** [comment]
see overleaf comments

File: backend/tests/image_review_scheduler.rs (hunk 2 above)
Comment (uglrl) [reply backend/tests/image_review_scheduler.rs:39]: I think it its cleaner this way

**Pull request merged**